### PR TITLE
[CCFPCM-380] Third Report page

### PR DESCRIPTION
**Pull request by chelsea-EYDS**
[CCFPCM-0380](https://bcdevex.atlassian.net/browse/CCFPCM-0380)

Objective: 

Third page CAS reporting

### CODE REVIEW

File: apps/backend/src/database/migrations/1679690933840-migration.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class migration1679690933840 implements MigrationInterface {
```
Comment (vyasworks) [inline apps/backend/src/database/migrations/1679690933840-migration.ts:3]: Let's add comments on why this migration needs to exist for future search and reference.

File: apps/backend/src/database/migrations/1679699038149-migration.ts [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class migration1679699038149 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`INSERT INTO master_location_data
+        (id, source_id, location_id, "method", pt_location_id, description, program_code, program_desc, ministry_client, resp_code, service_line_code, stob_code, project_code, merchant_id)
+
+
+      VALUES(uuid_generate_v4(), 'SBC', 10, 'Bank', 0, 'BURNABY', '70', 'BURNABY', '74', '32L10', '58200', '1461', '3200000', 999999999);`);
```
Comment (vyasworks) [inline apps/backend/src/database/migrations/1679699038149-migration.ts:9]: Same comment as above. 

Bonus > Make it a template and put it in the readme under migrations or appropriate place.

**Review by vyasworks — APPROVED** [review]
remove report xlsx.

File: apps/backend/src/database/migrations/1679618910864-migration.ts [hunk 3]
```diff
@@ -0,0 +1,175 @@
...
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "transaction" ALTER COLUMN "total_transaction_amount" TYPE numeric(16,2)`
+    );
+    await queryRunner.query(
+      `ALTER TYPE "public"."pos_deposit_status_enum" RENAME TO "pos_deposit_status_enum_old"`
+    );
+    await queryRunner.query(
+      `CREATE TYPE "public"."pos_deposit_status_enum" AS ENUM('PENDING', 'IN_PROGRESS', 'MATCH', 'EXCEPTION', 'ALL')`
```
[the remark this answers is not in the document]
Comment (chelsea-EYDS) [reply apps/backend/src/database/migrations/1679618910864-migration.ts]: removed all from enums

File: docs/migrations.md [hunk 4]
```diff
@@ -0,0 +1,6 @@
+#### Migrations Tracker:
+
+[1679618910864-migration](https://github.com/bcgov/PaymentCommonComponent/blob/9ba59fecf488e3779ad63a08d876f9941c8e1da3/apps/backend/src/database/migrations/1679618910864-migration.ts)
```
Comment (vyasworks) [inline docs/migrations.md]: I mean more like a template on what to include in each migration relating to functional changes, but this can be done later.

**Review by vyasworks — APPROVED** [review]

**Pull request merged**